pkg/jira/client.go

```diff
@@ -96,6 +96,7 @@ type Config struct {
 	Server   string
 	Login    string
 	APIToken string
+	Bearer   *bool
```

Comment:
Enum types would be better for this usecase instead of a boolean flag. This will make it easy to introduce other auth types if needed.

```go
type AuthType string

const (
	AuthTypeBasic  AuthType = "basic"
	AuthTypeBearer AuthType = "bearer"
)
```

And the `config` and `client` struct downstream can have `AuthType` field